clients/client-python/gravitino/api/expressions/sorts/sort_direction.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from enum import Enum
+from gravitino.api.expressions.sorts.null_ordering import NullOrdering
+
+
+class SortDirection(Enum):
+    """A sort direction used in sorting expressions.
+    Each direction has a default null ordering that is implied if no null ordering is specified explicitly.
+    """
+
+    ASCENDING = ("asc", NullOrdering.NULLS_FIRST)
+    """Ascending sort direction. Nulls appear first. For ascending order, this means nulls appear at the beginning."""
+
+    DESCENDING = ("desc", NullOrdering.NULLS_LAST)
+    """Descending sort direction. Nulls appear last. For ascending order, this means nulls appear at the end."""
+
+    _direction: str
+    _default_null_ordering: NullOrdering
```

Comment:
These lines should be removed.

Reply:
I know that Python can derive member variables, but I feel like showing all member variables explicitly. It will make the code easier to read.

Reply:
I know that Python can derive member variables, but I feel like showing all member variables explicitly. It will make the code easier to read.